### PR TITLE
shop/supermarket/IGA add wikidata and wikipedia tags

### PR DESCRIPTION
**Pull request by mraguso2**
Added tags for IGA Supermarket

### CODE REVIEW

File: config/canonical.json [hunk 1]
```diff
@@ -30100,6 +30100,8 @@
     "count": 619,
     "tags": {
       "brand": "IGA",
+      "brand:wikidata": "Q5970945",
+      "brand:wikipedia": "en:IGA_(supermarkets)",
```
Comment (bhousel) [inline config/canonical.json]: Remove the `_`, then it's good to merge.. 

```suggestion
      "brand:wikipedia": "en:IGA (supermarkets)",
```

**Comment by mraguso2** [comment]
removed "_" from IGA entry
I'm working on another entry that has a "_" in the wikipedia tag - should I just not include underscores?

**Comment by andrewharvey** [comment]
The wikipedia article and the wikidata are pointing to different things...? The wikidata is for the Australian IGA, the wikipedia is for the global IGA.

**Comment by bhousel** [comment]
> The wikipedia article and the wikidata are pointing to different things...? The wikidata is for the Australian IGA, the wikipedia is for the global IGA.

Thanks for noticing @andrewharvey - I just updated the wikidata tag in e2ee17b

**Pull request merged**